admin_interface/locale/fr/LC_MESSAGES/django.po

```diff
@@ -174,10 +178,22 @@ msgstr "code"
 msgid "display"
 msgstr "affichage"
 
+#: admin_interface/models.py
+msgid "foreground color"
+msgstr "couleur du premier plan (gras, champs requis, messages d'erreur)"
```

Comment:
It doesn’t seem correct to have this more detail in the translation and not the source strings: the English version will miss the details, and the other translators too!

Reply:
So what's the play here? Should I add the details in the verbose name in the models.py file and then edit this one?
Sorry this is my first time contributing on langage :/

Reply:
You should just translate the original message without adding extra informations, eg. `msgstr "couleur du premier plan"`

Reply:
Or add the full message in English in the source code!

Reply:
I went for an intermediate solution and instead I extended the help text. 
Instead of adding details in the verbose name, I felt that these details belong more to the help text.

Also added the translation for those new help texts.